ocs_ci/deployment/hosted_cluster.py

```diff
@@ -62,6 +62,25 @@
 logger = logging.getLogger(__name__)
 
 
+def skip_if_not_hcp_provider(func):
```

Comment:
Most of skip decorators are prefixed with skipif_ instead of skip_if_ and they are located in [ocs_ci/framework/pytest_customization/marks.py](https://github.com/red-hat-storage/ocs-ci/blob/46abf29c9f60591651997ebc51f2e538df3d68d7/ocs_ci/framework/pytest_customization/marks.py).

Reply:
this is not a mark, this is for skipping functions but not tests